test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -28,6 +28,194 @@ public PackCommandTests(MsbuildIntegrationTestFixture fixture)
+
+                var projectFileRelativ = Path.Combine(projectAndReference1Folder, projectName, $"{projectName}.csproj");
+                var referencedProject1RelativDir = Path.Combine(projectAndReference1Folder, referencedProject1, $"{referencedProject1}.csproj");
+                var referencedProject2RelativDir = Path.Combine(reference2Folder, referencedProject2, $"{referencedProject2}.csproj");
+
+                msbuildFixture.CreateDotnetNewProject(Path.Combine(testDirectory.Path, projectAndReference1Folder), projectName, "classlib");
+                msbuildFixture.CreateDotnetNewProject(Path.Combine(testDirectory.Path, projectAndReference1Folder), referencedProject1, "classlib");
+                msbuildFixture.CreateDotnetNewProject(Path.Combine(testDirectory.Path, reference2Folder), referencedProject2, "classlib");
+
+                msbuildFixture.RunDotnet(testDirectory.Path, $"new solution -n {solutionName}");
+                msbuildFixture.RunDotnet(testDirectory.Path, $"sln {solutionName}.sln add {projectFileRelativ}");
+                msbuildFixture.RunDotnet(testDirectory.Path, $"sln {solutionName}.sln add {referencedProject1RelativDir}");
+                msbuildFixture.RunDotnet(testDirectory.Path, $"sln {solutionName}.sln add {referencedProject2RelativDir}");
+
+                var projectFile = Path.Combine(testDirectory.Path, projectFileRelativ);
+                using (var stream = new FileStream(projectFile, FileMode.Open, FileAccess.ReadWrite))
+                {
+                    var xml = XDocument.Load(stream);
+                    var attributes = new Dictionary<string, string>();
+                    var properties = new Dictionary<string, string>();
+
+                    ProjectFileUtils.AddItem(xml, "ProjectReference", @"..\ClassLibrary2\ClassLibrary2.csproj", string.Empty, properties, attributes);
+                    ProjectFileUtils.AddItem(xml, "ProjectReference", @"..\ClassLibrary3\ClassLibrary3.csproj", string.Empty, properties, attributes);
+
+                    ProjectFileUtils.WriteXmlToFile(xml, stream);
+                }
+
+                msbuildFixture.RestoreSolution(testDirectory, solutionName, string.Empty);
+
+                var nupkgPath = Path.Combine(projectFolder, @"bin\Debug", $"{projectName}.1.0.0.nupkg");
+                var nuspecPath = Path.Combine(projectFolder, @"obj\Debug", $"{projectName}.1.0.0.nuspec");
+
+                // Act
+                msbuildFixture.PackSolution(testDirectory, solutionName, string.Empty, null);
+
+                // Assert
+                Assert.True(File.Exists(nupkgPath), "The output .nupkg is not in the default place");
+                Assert.True(File.Exists(nuspecPath), "The intermediate nuspec file is not in the default place");
+            }
+        }
+
+        [PlatformFact(Platform.Windows)]
+        public void PackCommand_NewSolution_ContinuousOutputInBothDefaultAndCustomPaths()
```

Comment:
nit: The `In both` part confused me, made me think it was outputing in both location, which is obviously not something we're doing :)

Reply:
It is output'ing in both locations but not at the same time. It's like this, when you have a project, you build with defaults to see it succeed and then you run an official build to publish the artifacts in a different folder using different config without affecting the previous build. A common dev scenario (at least, in our case!!).

The name is little confusing, I agree but if you have an alternative, we could use that instead. I'll queue up the changes in the next PR if it's alright with you!?

Reply:
I don't have a great suggestion, I'm fine with leaving it be.